frontend/src/components/gradebook/gradebook.scss

```diff
@@ -61,9 +61,37 @@
     border: 1px solid lightgray;
 
     padding: 0.5em 1em;
+    font-size: 16px;
+
+    .expand {
+      color: #FFC904;
+      font-size: 12px;
+      cursor: pointer;
+      text-shadow: 1px 1px gray;
+    }
+
+    .date {
+      font-size: 10px;
```

Comment:
This is super minor, but we shouldn't go below 12px fonts.

Reply:
That's the font size on the design doc :<

Reply:
I know, but for accessibility it is not good.